Source/WebCore/accessibility/isolatedtree/AXIsolatedTree.cpp

```diff
@@ -987,6 +987,17 @@ RefPtr<AXIsolatedObject> AXIsolatedTree::rootNode()
     return m_rootNode;
 }
 
+RefPtr<AXIsolatedObject> AXIsolatedTree::webArea()
```

Comment:
Maybe `rootWebArea()` instead of `webArea()`? Since technically there can be more than one in a non-site-isolated WebKit when `iframes` are on the page.

Reply:
Good point, then it will match what we have in `AXObjectCache` too